cmd/podman/generate/systemd.go

```diff
@@ -119,6 +127,7 @@ func init() {
 }
 
 func systemd(cmd *cobra.Command, args []string) error {
+	fmt.Fprint(os.Stderr, deprecation)
```

Comment:
Do we really need to print this? I am personally not the biggest fan of the warnings. I mean its great to get the message out but it is also very annoying.
Not a blocker for me just saying.

Reply:
It's indeed very annoying but I think it may be worth giving a shot.  I think that many users are still not familiar with Quadlet and that's a way to let them know about it - very loudly :)

In case users bark back, we can revert.